observatory-platform/observatory/platform/docker/Dockerfile.observatory.jinja2

```diff
@@ -14,7 +14,7 @@
 
 # Author: James Diprose
 
-FROM apache/airflow:1.10.14-python3.7
+FROM apache/airflow:2.1.2-python3.8
```

Comment:
It would be good if the base Airflow Docker image name / version was a parameter

Reply:
I might do this as part of the work on installation scripts instead since there's already some work done there on templating out versions